testing/next/APKBUILD

```diff
@@ -0,0 +1,24 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Benjamin Buccianti <bebuccianti@gmail.com>
+pkgname=next
+pkgver=1.3.2
+pkgrel=0
+pkgdesc="Atlas Engineer Next Browser"
+url="https://next.atlas.engineer"
+arch="x86_64"
+license="BSD-3-Clause"
+source="https://github.com/atlas-engineer/next/archive/$pkgver.zip"
```

Comment:
I think you should rename the tarball here

Reply:
This I can do, you'll see a fix inbound for the line break as well.